lib/airbrake.js

```diff
@@ -138,61 +138,61 @@ Airbrake.prototype._sendRequest = function(err, cb) {
     headers: {
       'Content-Length': body.length,
       'Content-Type': 'application/json',
-      'Accept': 'application/json',
-    },
+      Accept: 'application/json'
+    }
   }, this.requestOptions);
 
-  request(options, function(err, res, body) {
-    if (err) {
+  request(options, function (requestErr, res, responseBody) {
+    if (requestErr) {
       return callback(err);
     }
 
-    if (undefined === body) {
+    if (typeof responseBody === 'undefined') {
```

Comment:
Normally, in JS land I believe it is enough to just allow falsy values to be evaluated in the conditional like so:
```js
if (!responseBody) {
  return callback(new Error('invalid body'));
}
```

However, I've been trying to maintain a similar style to the author's original check. The order of evaluation looked weird, and for the most part in JS land I think people use `typeof x === 'undefined'` so you don't have to evaluate the type twice.